### ISSUE TITLE
Rapidvideo down

### ISSUE THREAD

**Issue by bjgood** [jsergio123/script.module.resolveurl#89]
Hi JSergio,
Rapidvideo is down. It has been for the last two or three days. This is one of the good ones.

Thought I report it. Hope you return soon.

Have a good one. :).

**Comment by Gujal00** [comment]
Rapidvideo recently changed their certificate provider as cloudflare dumped them.
Kodi 18.1 comes with update certstore and rapidvideo works in Kodi18.1

**Comment by solbu** [comment]
> Rapidvideo recently changed their certificate provider as cloudflare dumped them.
> Kodi 18.1 comes with update certstore and rapidvideo works in Kodi18.1

Why should Kodi care?
A changed provider should already be in the major OS'es certificate stores already. And the os cert stores are populated by the major browsers.

There is an OS certificate store for a reason.

**Comment by Gujal00** [comment]
Yeah tell that to mozilla who maintain their trust store independently of any OS and a static extract of which is bundled with kodi

**Comment by doko-desuka** [comment]
If you replace the https:// with http:// the media is served without any certificate.

**Edited:** ---  

@bjgood can you try changing this line ([#57](https://github.com/jsergio123/script.module.resolveurl/blob/master/lib/resolveurl/plugins/rapidvideocom.py#L57)) of rapidvideocom.py, from this...  
`return helpers.pick_source(sources) + helpers.append_headers(headers)`  

to this...  

```
return helpers.pick_source(sources).replace('https://', 'http://', 1) + helpers.append_headers(headers)
```

**Comment by bjgood** [comment]
Thanks everyone for the replies.

I run LibreELEC 9 (official release) on a RPi3. I installed Milhouse's Leia build #0220, Kodi (18.2-RC1 Git:d21fed4), released yesterday, but rapidvideo still doesn't show during a scan.

And, doko, I tried your code change in rapidvideocom.py, but no change in results.

Thanks for your interest and efforts everyone.

Have a good one. :). At least it isn't raining today where I live.

Later...
